salt/utils/event.py

```diff
@@ -444,9 +475,16 @@ def get_event(self, wait=5, tag='', full=False, match_type=None):
              - 'endswith' : search for event tags that end with tag
              - 'find' : search for event tags that contain tag
              - 'regex' : regex search '^' + tag event tags
+             - 'fnmatch' : fnmatch tag event tags matching
             Default is opts['event_match_type'] or 'startswith'
 
-            .. versionadded:: Boron
+            .. versionadded:: Beryllium
+
+        no_block
+            Define if getting the event should be a blocking call or not.
+            Defaults to True to keep backwards compatibility.
+
+            .. versionadded:: Beryllium
```

Comment:
It used to be blocking no? Even says so in the code:
https://github.com/saltstack/salt/pull/26880/files#diff-457a19731b8307d768e8fc84a9ba28e8L406
Before this PR, it calls "self.get_event_block()"
After this PR, it now by default calls "self.get_event_noblock()"

Maybe it was my comment in `get_returns_no_block` that caused confusion. Apologies.
That comment was referring to itself, `get_returns_no_block`, which WAS previously non blocking (it used to call `get_event_noblock` directly and duplicate a lot of code, I changed it to use `get_event` so it could take advantage of the new caching, since everywhere else was using `get_event`, which in turn made it blocking)
https://github.com/s0undt3ch/salt/commit/65acf975dd03ea44fe366304ccfe35ae12050fb3#diff-afe7362c6e0817c26aaf1e3c4351ba97L829